### PR TITLE
Fix grammar and incorrect output messages

### LINKED ISSUE #201: [PE-D][Tester A] Error Message of incorrect Tag format has formatting issue
In the error message given, "ofalphabets" should be "of alphabets"

-------------
Labels: `type.FunctionalityBug` `severity.VeryLow`
original: Apzure/ped#3

### LINKED ISSUE #206: Incorrect grammar in output message of find command
The command executed is `find alex` on the default book.
The message should be `1 person listed` instead of `1 persons listed`.

image

### LINKED ISSUE #221: [PE-D][Tester E] UI display text is missing a space
## Description
As I was keying in an invalid asset command, the error message shown seemed to be missing a space between the words. On a separate occasion, as I was keying in adding an invalid tag to a contact, the error message shown seemed to be missing a space between the words. 

## Possible Solution
Perhaps it would be good to double-check how the error messages are displayed to the users.

-------------
Labels: `severity.VeryLow` `type.FeatureFlaw`
original: triciiaaa/ped#9

### PR DESCRIPTION
**Pull request by bryanyee33**
Closes #201 
Closes #206 
Closes #221 

Allowed to change "person" -> "contact": https://github.com/nus-cs2103-AY2324S2/forum/issues/775

### CODE REVIEW

File: src/main/java/seedu/address/logic/Messages.java [hunk 1]
```diff
@@ -15,14 +15,14 @@ public class Messages {
...
+    public static final String MESSAGE_PERSONS_LISTED_OVERVIEW = "%1$d person(s) listed!";
     public static final String MESSAGE_DUPLICATE_FIELDS =
                 "Multiple values specified for the following single-valued field(s): ";
 
     public static final String MESSAGE_SHOWING_HELP = "Opened help window.";
     public static final String MESSAGE_EXITING = "Exiting Address Book as requested ...";
-    public static final String MESSAGE_COPIED = "Copied details to clipboard: %1$s";
-    public static final int MESSAGE_COPIED_LEN = "Copied details to clipboard:".length();
+    public static final String MESSAGE_COPIED = "Copied detail to clipboard: %1$s";
+    public static final int MESSAGE_COPIED_LEN = "Copied detail to clipboard:".length();
```
Comment (aureliony) [inline src/main/java/seedu/address/logic/Messages.java:25]: Should this really be changed?

File: src/main/java/seedu/address/model/person/fields/Phone.java [hunk 2]
```diff
@@ -12,7 +12,7 @@
 public class Phone implements Field {
 
     public static final Prefix PREFIX_PHONE = new Prefix("p\\");
-    private static final String MESSAGE_CONSTRAINTS = "Phone numbers should only contain numbers,"
+    private static final String MESSAGE_CONSTRAINTS = "Phone numbers should only contain numbers, "
             + "'+', '-', ',' and spaces, and it should be at least 3 characters long";
```
Comment (aureliony) [inline src/main/java/seedu/address/model/person/fields/Phone.java:16]: Should we add a whitespace between each symbol to the message?

**Review by aureliony — CHANGES_REQUESTED** [review]

File: src/main/java/seedu/address/logic/Messages.java (hunk 1 above)
Comment (bryanyee33) [reply src/main/java/seedu/address/logic/Messages.java:25]: I'll change it to detail(s), as it can be either singular or plural based on the field.

**Review by yisiox — APPROVED** [review]
LGTM

**Review by aureliony — APPROVED** [review]
LGTM

**Pull request merged**